airflow/cli/commands/jobs_command.py

```diff
@@ -32,7 +32,11 @@ def check(args, session: Session = NEW_SESSION) -> None:
     if args.hostname and args.local:
         raise SystemExit("You can't use --hostname and --local at the same time")
 
-    query = session.query(Job).filter(Job.state == State.RUNNING).order_by(Job.latest_heartbeat.desc())
+    query = (
+        session.query(Job)
+        .filter(Job.state == TaskInstanceState.RUNNING)
```

Comment:
this one's a bit awkward... 
does it make sense to use TI state here?

Reply:
The value is mostly forwarded from TaskInstance, but I do wonder about the same. Maybe a new JobState makes sense?

Reply:
Probably? But I don't mind either way